components/eventing-controller/api/v1alpha1/subscription_types.go

```diff
@@ -184,6 +184,10 @@ type SubscriptionStatus struct {
 	// Ready defines the overall readiness status of a subscription
 	Ready bool `json:"ready"`
 
+	// CleanEventTypes defines the clean filter event types
```

Comment:
CleanEventTypes defines the filter event types after cleanup for use with the configured backend